specs/language/statements.tex

```diff
@@ -1,2 +1,93 @@
 \Ch{Statements}{Stmt}
 \Sec{Label Statements}{Stmt.Label}
+\Sec{Attributes}{Stmt.Attr}
+\Sub{Unroll Attribute}{Stmt.Attr.Unroll}
+The \texttt{[unroll]} and \texttt{[unroll(n)]} attribute qualifiers can be used
```

Comment:
nit: the term qualifiers has a specific meaning in C++, so we should avoid overloading its use. I think we can also term this as the _unroll_ attribute, the argument doesn't actually change what attribute it is, it just changes the behavior.

```suggestion
The \textit{unroll} attribute can be used
```